dll/cpl/intl/lang/en-US.rc

```diff
@@ -214,3 +214,8 @@ BEGIN
     IDS_ERROR_NLS_CODE_REG "Problem opening key: HKLM\\SYSTEM\\CurrentControlSet\\Control\\NLS\\CodePage"
     IDS_ERROR_INPUT_DLL "Unable to start input.dll"
 END
+
+STRINGTABLE
```

Comment:
Any reason to add a new table rather than including this string in the previous one?

Reply:
No specific reason. Would you prefer the previous table ? Another one ? What's the impact ?